lib/services/runtime/handlers/noMatch.ts

```diff
@@ -44,12 +46,16 @@ const getOutput = (runtime: Runtime, node: NoMatchNode, noMatchCounter: number,
   const sanitizedVars = sanitizeVariables(variables.getState());
   const globalNoMatchPrompt = getGlobalNoMatchPrompt(runtime);
 
-  if (exhaustedReprompts) {
-    return replaceVariables(globalNoMatchPrompt?.content, sanitizedVars);
+  if (!exhaustedReprompts) {
+    const speak = (node.noMatch?.randomize ? _.sample(noMatchPrompts) : noMatchPrompts?.[noMatchCounter]) || '';
+    return replaceVariables(speak, sanitizedVars);
   }
 
-  const speak = (node.noMatch?.randomize ? _.sample(noMatchPrompts) : noMatchPrompts?.[noMatchCounter]) || '';
-  return replaceVariables(speak, sanitizedVars);
+  if (!isPromptContentInitialyzed(globalNoMatchPrompt?.content)) {
```

Comment:
```suggestion
  if (!isPromptContentInitialized(globalNoMatchPrompt?.content)) {
```

Whoops, spelling error. But it's in general-runtime, so kinda lost cause 😆

Reply:
oh 💩 , my bad.